### PR TITLE
Condense README examples

### PR DESCRIPTION
**Pull request by asl3**
Remove README.md examples other than `transform` and `plot` to improve readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,28 +127,6 @@ Now when you call df.ai.transform as before, the agent will use word embeddings
...
-|------|-----------|---------------|----------------------|
-| 1    | Toyota    | 1849751       | -9                   |
-| 2    | Ford      | 1767439       | -2                   |
-| 3    | Chevrolet | 1502389       | 6                    |
-| 4    | Honda     | 881201        | -33                  |
-| 5    | Hyundai   | 724265        | -2                   |
-
 ### Plot
 ```python
 auto_df.ai.plot()
```
Comment (gengliangwang) [inline README.md:132]: Let's add one line for the creation of `auto_df`

e.g
```
# auto sales data from https://www.carpro.com/blog/full-year-2022-national-auto-sales-by-brand
data = [('Toyota', 1849751, -9), ('Ford', 1767439, -2), ('Chevrolet', 1502389, 6),
        ('Honda', 881201, -33), ('Hyundai', 724265, -2), ('Kia', 693549, -1),
        ('Jeep', 684612, -12), ('Nissan', 682731, -25), ('Subaru', 556581, -5),
        ('Ram Trucks', 545194, -16), ('GMC', 517649, 7), ('Mercedes-Benz', 350949, 7),
        ('BMW', 332388, -1), ('Volkswagen', 301069, -20), ('Mazda', 294908, -11),
        ('Lexus', 258704, -15), ('Dodge', 190793, -12), ('Audi', 186875, -5),
        ('Cadillac', 134726, 14), ('Chrysler', 112713, -2), ('Buick', 103519, -42),
        ('Acura', 102306, -35), ('Volvo', 102038, -16), ('Mitsubishi', 102037, -16),
        ('Lincoln', 83486, -4), ('Porsche', 70065, 0), ('Genesis', 56410, 14),
        ('INFINITI', 46619, -20), ('MINI', 29504, -1), ('Alfa Romeo', 12845, -30),
        ('Maserati', 6413, -10), ('Bentley', 3975, 0), ('Lamborghini', 3134, 3),
        ('Fiat', 915, -61), ('McLaren', 840, -35), ('Rolls-Royce', 460, 7)]

auto_df = spark.createDataFrame(data, ["Brand", "US_Sales_2022", "Sales_Change_Percentage"])
```

Comment (asl3) [reply README.md:132]: makes sense, in that case shall we just keep the original data ingestion section? to demonstrate ingestion from URL?

Comment (gengliangwang) [reply README.md:132]: Yes we can, but it requires extra setup. Note that the ingestion will be truncated if the number of rows is larger than 30.
Or, we can put 
```
('Toyota', 1849751, -9), ('Ford', 1767439, -2), ('Chevrolet', 1502389, 6),
        ('Honda', 881201, -33), ('Hyundai', 724265, -2), ('Kia', 693549, -1),
        ('Jeep', 684612, -12), ('Nissan', 682731, -25), ('Subaru', 556581, -5),
        ('Ram Trucks', 545194, -16), ('GMC', 517649, 7), ('Mercedes-Benz', 350949, 7),
        ('BMW', 332388, -1), ('Volkswagen', 301069, -20), ('Mazda', 294908, -11),
        ('Lexus', 258704, -15), ('Dodge', 190793, -12), ('Audi', 186875, -5),
        ('Cadillac', 134726, 14), ('Chrysler', 112713, -2), ('Buick', 103519, -42),
        ('Acura', 102306, -35), ('Volvo', 102038, -16), ('Mitsubishi', 102037, -16),
        ('Lincoln', 83486, -4), ('Porsche', 70065, 0), ('Genesis', 56410, 14),
        ('INFINITI', 46619, -20), ('MINI', 29504, -1), ('Alfa Romeo', 12845, -30),
        ('Maserati', 6413, -10), ('Bentley', 3975, 0), ('Lamborghini', 3134, 3),
        ('Fiat', 915, -61), ('McLaren', 840, -35), ('Rolls-Royce', 460, 7)
```
into a csv file and load the csv file, either way is fine.

Comment (asl3) [reply README.md:155]: ah ok, maybe it's best to keep it simple for now and just go with your first comment:

`auto_df = spark.createDataFrame(data, ["Brand", "US_Sales_2022", "Sales_Change_Percentage"])`

File: README.md [hunk 2]
```diff
@@ -127,30 +127,31 @@ Now when you call df.ai.transform as before, the agent will use word embeddings
...
+        ('BMW', 332388, -1), ('Volkswagen', 301069, -20), ('Mazda', 294908, -11),
+        ('Lexus', 258704, -15), ('Dodge', 190793, -12), ('Audi', 186875, -5),
+        ('Cadillac', 134726, 14), ('Chrysler', 112713, -2), ('Buick', 103519, -42),
+        ('Acura', 102306, -35), ('Volvo', 102038, -16), ('Mitsubishi', 102037, -16),
+        ('Lincoln', 83486, -4), ('Porsche', 70065, 0), ('Genesis', 56410, 14),
+        ('INFINITI', 46619, -20), ('MINI', 29504, -1), ('Alfa Romeo', 12845, -30),
+        ('Maserati', 6413, -10), ('Bentley', 3975, 0), ('Lamborghini', 3134, 3),
+        ('Fiat', 915, -61), ('McLaren', 840, -35), ('Rolls-Royce', 460, 7)]
+
+auto_df = spark.createDataFrame(data, ["Brand", "US_Sales_2022", "Sales_Change_Percentage"])
```
Comment (gengliangwang) [inline README.md:148]: where is `spark` from? We can create a new SparkSession

Comment (asl3) [reply README.md]: shall we match the previous ingestion example:

`auto_df = spark_ai._spark.createDataFrame(..)`

**Review by gengliangwang — APPROVED** [review]

**Pull request merged**